.github/workflows/publish_to_dockerhub.yml

```diff
@@ -43,3 +43,30 @@ jobs:
       - name: Push Prod Tags
         if: steps.check-tag.outputs.match == 'true'
         run: nox -s "push(prod)"
+  push-fidesops-privacy-center-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ env.DOCKER_USER }}  # Needs updating for fidesops privacy center
+          password: ${{ env.DOCKER_TOKEN }}  # Needs updating for fidesops privacy center
```

Comment:
These will need to be updated for `ethyca/fidesops-privacy-center` on docker hub